### PR TITLE
2017 q1.refactoring.canfile.length

### PR DESCRIPTION
**Pull request by fgiaimo**
To be merged only right after https://github.com/se-research/OpenDaVINCI/pull/165

### CODE REVIEW

**Review by chrberger — APPROVED** [review]
Did you manually transform this file?

**Comment by fgiaimo** [comment]
Yes. Whatever automated tool we had before, it will need to be updated. I may try to do something about it in the future.

File: resources/can/fh16mapping.can [hunk 1]
```diff
@@ -9,54 +9,66 @@ using ODVDFH16Truck;
...
-wheelspeeds1.driveaxle1wheelspeedright in 0x101 at bit 48 for 16 bit is unsigned little endian multiply by 0.00390625 add 0 with range [0, 255.99609375];
-wheelspeeds1.driveaxle1wheelspeedleft in 0x101 at bit 32 for 16 bit is unsigned little endian multiply by 0.00390625 add 0 with range [0, 255.99609375];
-wheelspeeds1.frontaxle1wheelspeedright in 0x101 at bit 16 for 16 bit is unsigned little endian multiply by 0.00390625 add 0 with range [0, 255.99609375];
-wheelspeeds1.frontaxle1wheelspeedleft in 0x101 at bit 0 for 16 bit is unsigned little endian multiply by 0.00390625 add 0 with range [0, 255.99609375];
-
+CAN Message 0x102 wheelspeeds2 : 4 {
+    driveaxle2wheelspeedright at bit 16 for 16 bit is unsigned little endian multiply by 0.00390625 add 0 with range [0, 255.99609375];
+    driveaxle2wheelspeedleft at bit 0 for 16 bit is unsigned little endian multiply by 0.00390625 add 0 with range [0, 255.99609375];
+}
+CAN Message 0x101 wheelspeeds1 : 8 {
```
Comment (chrberger) [inline resources/can/fh16mapping.can]: @fgiaimo Could you improve the language design slightly:

CAN Message 0x101 wheelspeeds1 : 8 {
-->
CANMessage 0x0101 wheelspeeds1 is 8 bytes {

**Review by chrberger — CHANGES_REQUESTED** [review]

**Review by chrberger — CHANGES_REQUESTED** [review]
This PR should be merged when the script to handle .dbc -> .can is ready and this branch has been verified on the truck again (using OpenDLV.rhino).

**Pull request merged**